src/doc/unstable-book/src/compiler-flags/coverage-options.md

```diff
@@ -5,4 +5,7 @@ This option controls details of the coverage instrumentation performed by
 
 Multiple options can be passed, separated by commas. Valid options are:
 
-- `no-branch`, `branch` or `mcdc`: `branch` enables branch coverage instrumentation and `mcdc` further enables modified condition/decision coverage instrumentation. `no-branch` disables branch coverage instrumentation as well as mcdc instrumentation, which is same as do not pass `branch` or `mcdc`.
+- `block` (default): Disables branch coverage and MC/DC coverage.
+- `branch`: Enables branch coverage instrumentation.
+- `mcdc`: Enables Modified Condition/Decision Coverage instrumentation.
```

Comment:
We should explicitely emphasize that `branch` implies`block` and `mcdc` implies `branch`.